src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/DeleteRecommendationService.kt

```diff
@@ -43,12 +43,12 @@ internal class DeleteRecommendationService(
   suspend fun getSystemDeleteRecommendationResponse(
     crn: String,
   ): ResponseEntity<DeleteRecommendationResponse> {
-    val recommendations = recommendationRepository.findByCrn(crn).sorted()
+    val recommendations = recommendationRepository.findByCrn(includeDeleted = true, crn = crn).sorted()
     return ResponseEntity(
       DeleteRecommendationResponse(
-        sensitive = recommendations.firstOrNull()?.data?.sensitive ?: false,
+        sensitive = recommendations[0].data.sensitive ?: false,
```

Comment:
Isn't this going to throw an exception if recommendations is empty?

Reply:
The crn comes from a prior database call in RecommendationsCleanupTask so at this point we already know it exists.